website/docs/module_slack.md

```diff
@@ -69,8 +81,22 @@ Few keys also have Capitalized and UPPERCASE format. Values will be modified acc
 **Available keys:**
```

Comment:
I would rather point to anatomy data and here add those keys that are not in anatomy data. It is hell to track all docs where these keys are when something in anatomy changes.